public/docs/ts/latest/cookbook/angular-in-dotnet-core.jade

```diff
@@ -0,0 +1,290 @@
+include ../_util-fns
+
+:marked
+  This cookbook describes how to build an Angular app inside the MVC architecture in a .NET Core Web application. It does not explain how Angular or .NET Core works. However it will provide all the code snippets required. Therefore you will be able to follow through this cookbook even if you are currently not familiar with Angular or .NET Core.
+
+  It is recommended that you follow this cookbook step by step and experience the challenges in the process.
+
+  .NET Core supports cross platforms. You can develop .NET Core applications on Windows, macOS or Linux. This cookbook will use Visual Studio 2015 on Windows.
```

Comment:
Suggest ... 

> .NET Core is a cross platform solution.

Saying the cookbook uses Windows makes me wonder what will be different on OSX on a Mac. I think it would be good to discuss both in the cookbook, otherwise we cut out a growing audience.